akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -208,13 +209,24 @@ class ClusterSharding(system: ExtendedActorSystem) extends Extension {
     allocationStrategy: ShardAllocationStrategy,
     handOffStopMessage: Any): ActorRef = {
 
-    requireClusterRole(settings.role)
-    implicit val timeout = system.settings.CreationTimeout
-    val startMsg = Start(typeName, entityProps, settings,
-      extractEntityId, extractShardId, allocationStrategy, handOffStopMessage)
-    val Started(shardRegion) = Await.result(guardian ? startMsg, timeout.duration)
-    regions.put(typeName, shardRegion)
-    shardRegion
+    if (settings.shouldHostShard(cluster)) {
+
+      implicit val timeout = system.settings.CreationTimeout
+      val startMsg = Start(typeName, entityProps, settings,
+        extractEntityId, extractShardId, allocationStrategy, handOffStopMessage)
+      val Started(shardRegion) = Await.result(guardian ? startMsg, timeout.duration)
+      regions.put(typeName, shardRegion)
+      shardRegion
+    } else {
+      system.log.info("Starting Shard Region Proxy [{}] (no actors will be hosted on this node)...", typeName)
+
+      startProxy(
+        typeName,
+        settings.role,
+        dataCenter = None, // TODO what about the multi-dc value here?
```

Comment:
Hmm good catch, here perhaps leave none, but we need to add an override to start that would take the datacenter?

cc @patriknw

Reply:
I see that currently there is no ability to start the shard region itself in a remote datacenter. So, probably, in this case it also doesn't make sense (i.e. probably we can just leave `None` here).

Reply:
You can't start sharding on someone's nodes, yeah :) 
But you can proxy to sharding that's running somewhere else, so it should be possible I guess

Reply:
As I can see this: Here we're trying to start the shard region and in case of a role mismatch (like a fallback case) - then create a proxy. As we have intent to *start* the shard region, then this method should be usable only for local datacenter, so, the `dataCenter` parameter probably is useless and we can just leave `None` if we're starting a proxy as a fallback (instead of throwing an exception).

Reply:
I agree, it should be `None` here. If one need a proxy for another DC then the `startProxy` method must be used.